### ISSUE TITLE
Toolkit Process Builder: action box height being variable to content creates annoying user experience

### ISSUE THREAD

**Issue by jpailthorpe** [engagementgamelab/city-accelerator-website#88]
I would like to have the boxes on the builder tool set at a min height so the buttons don't continue to change positions between yes/no, and all content fits. The way I think we could accomplish this is determine what the maximum amount of space is needed out of all the content cards.  It creates a slightly annoying user experience, and was one of the bigger asks by stakeholders. I do agree it is quite frustrating. 

Of course this will change on tablet/mobile. Also open to a design solution that fixes this without setting min height, because it def. will be a little wonky.

**Comment by msuen** [comment]
Agreed. This was the originally intended design (i.e. all cards have the same height), but I failed to communicate this, so I apologize.

**Comment by MarianoMiguel** [comment]
@msuen @jpailthorpe Working on a solution for this one

**Comment by esalling23** [comment]
<img width="1196" alt="screen shot 2017-01-27 at 10 45 13 am" src="https://cloud.githubusercontent.com/assets/11821504/22376922/f853c026-e47d-11e6-8426-3039e3c6f48c.png">
Now seeing this after making a new QA build with the fix above!

**Comment by jpailthorpe** [comment]
Above error was fixed^. 

However, there is another issue where the height of the box is determined by the height of the content.

**Comment by esalling23** [comment]
Also seeing similar cut off of the main height on the MVP grid
<img width="755" alt="screen shot 2017-01-27 at 4 35 58 pm" src="https://cloud.githubusercontent.com/assets/11821504/22388488/bcf756e2-e4ae-11e6-9013-606dc1da2bf4.png">

**Comment by jpailthorpe** [comment]
Actually this is happening on many different loads. I feel like we might have gotten ourselves down a rabbit hole with this. Even if it works properly, it seems a bit weird that some boxes you have to scroll super far down (because there is more content) esp on mobile devices. Part of me wonders if we just have to revert back to the height being determined by each content box as it comes. Thoughts?
